code/game/machinery/_machinery.dm

```diff
@@ -152,6 +152,8 @@
 	if(occupant_typecache)
 		occupant_typecache = typecacheof(occupant_typecache)
 
+	ADD_TRAIT(src, TRAIT_PROTECT_FOOD, FOOD_MACHINE_TRAIT) // I realized if it's going to be inside ANY MACHINE it'd probably get preserved
```

Comment:
This feels a bit strange, why do we need to mark machines as explicitly ok? can't we limit the check to just the food being on a turf while  not on the same tile as a table of some sort?

Reply:
If that was the case, then placing food inside any machine that's not on a table would still start the timer, like smartfridges or microwaves that were moved.

Reply:
discussing this with mothblocks, please hold

Reply:
Said over discord I removed it, but I'll repeat that here for posterity.